drivers/volume/csi/csi.go

```diff
@@ -1058,6 +1118,14 @@ func (c *csi) restorePVC(
 	return pvc, nil
 }
 
+func (c *csi) getRestoreSnapshotName(pvc *v1.PersistentVolumeClaim, restore *storkapi.ApplicationRestore) string {
+	return fmt.Sprintf("%s-vs-%s-%s", snapshotRestorePrefix, string(restore.UID), pvc.Name)
```

Comment:
Since the name can be close to the max length limit this constructed name can exceed the limit.

Can we use just the last part of the UID of the restore and PVC to fix the size of the names?

Reply:
Yeah, I updated this in the PR. I also updated the px-backup deletion objects to use similar naming.